modules/case-management/src/test/java/com/ibm/cloud/platform_services/case_management/v1/CaseManagementIT.java

```diff
@@ -0,0 +1,367 @@
+        assertNotNull(response);
+    }
+
+    @Test
+    public void testCreateCase() {
+
+        String caseSubject = "Java - Integration test";
+        String description = "Please -ignore this is a test case.";
+
+        // Offering info can be retrieved via /case-management/utilities/v1/offerings/technical
+        OfferingPayloadType offeringPayloadTypeModel = new OfferingPayloadType.Builder()
+            .group("crn_service_name")
+            .key("cloud-object-storage")
+            .id("dff97f5c-bc5e-4455-b470-411c3edbe49c")
+            .build();
+
+        OfferingPayload offeringPayloadModel = new OfferingPayload.Builder()
+            .name("Cloud Object Storage")
+            .type(offeringPayloadTypeModel)
+            .build();
+
+        CreateCaseOptions createCaseOptionsModel = new CreateCaseOptions.Builder()
+            .type("technical")
+            .subject(caseSubject)
+            .description(description)
+            .offering(offeringPayloadModel)
+            .severity(Long.valueOf("4"))
+            .build();
+
+        Response<Case> response = service.createCase(createCaseOptionsModel).execute();
+
+        Case responseObj = response.getResult();
+
+        assertEquals(caseSubject, responseObj.getShortDescription());
+        assertEquals(description, responseObj.getDescription());
+
+        // Storing the new case number for subsequent test cases
+        this.newCaseNumber = responseObj.getNumber();
+
+        System.out.println("New case number: " + responseObj.getNumber());
+    }
+
+    @Test (dependsOnMethods = {"testCreateCase"})
+    public void testGetCases() {
+
+        String[] fields = new String[] {"number", "description"};
+
+        GetCasesOptions getCasesOptionsModel = new GetCasesOptions.Builder()
+            .offset(Long.valueOf("0"))
+            .limit(Long.valueOf("2"))
+            .sort("number")
+            .status(new ArrayList<String>(Arrays.asList("new")))
+            .fields(new ArrayList<String>(Arrays.asList(fields)))
+            .build();
+
+        Response<CaseList> response = service.getCases(getCasesOptionsModel).execute();
+        assertNotNull(response);
+        CaseList responseObj = response.getResult();
+
+        assertTrue(0 < responseObj.getTotalCount());
+        assertTrue(0 < responseObj.getCases().size());
+    }
+
+    @Test (dependsOnMethods = {"testCreateCase"})
+    public void testGetCase() {
+
+        String[] fields = new String[] {"number", "description"};
+
+        GetCaseOptions getCaseOptionsModel = new GetCaseOptions.Builder()
+            .caseNumber(newCaseNumber)
+            .fields(new ArrayList<String>(Arrays.asList(fields)))
+            .build();
+
+        Response<Case> response = service.getCase(getCaseOptionsModel).execute();
+        assertNotNull(response);
+        Case responseObj = response.getResult();
+        assertEquals(newCaseNumber, responseObj.getNumber());
```

Comment:
I think we should assert that the `description` is there and no other properties are there in the case response.

Reply:
Will do. Easy enough.